spec/jobs/resolution_proofing_job_spec.rb

```diff
@@ -335,8 +335,18 @@
 
         perform
 
+        result = document_capture_session.load_proofing_result[:result]
+        result_context = result[:context]
+        result_context_stages = result_context[:stages]
+        result_context_stages_threatmetrix = result_context_stages[:threatmetrix]
```

Comment:
good candidate for `.dig`?

```suggestion
        result_context_stages_threatmetrix = result.dig(:context, :stages, :threatmetrix)
```

Reply:
I went with this approach because it matches a pattern used in other test examples in this file.

Examples:

- https://github.com/18F/identity-idp/blob/3f9699ea16aa4a20eee7f23025b5e14b9f5b29da/spec/jobs/resolution_proofing_job_spec.rb#L131
- https://github.com/18F/identity-idp/blob/3f9699ea16aa4a20eee7f23025b5e14b9f5b29da/spec/jobs/resolution_proofing_job_spec.rb#L158
- https://github.com/18F/identity-idp/blob/3f9699ea16aa4a20eee7f23025b5e14b9f5b29da/spec/jobs/resolution_proofing_job_spec.rb#L196
- https://github.com/18F/identity-idp/blob/3f9699ea16aa4a20eee7f23025b5e14b9f5b29da/spec/jobs/resolution_proofing_job_spec.rb#L230
- https://github.com/18F/identity-idp/blob/3f9699ea16aa4a20eee7f23025b5e14b9f5b29da/spec/jobs/resolution_proofing_job_spec.rb#L264
- https://github.com/18F/identity-idp/blob/3f9699ea16aa4a20eee7f23025b5e14b9f5b29da/spec/jobs/resolution_proofing_job_spec.rb#L289
- https://github.com/18F/identity-idp/blob/3f9699ea16aa4a20eee7f23025b5e14b9f5b29da/spec/jobs/resolution_proofing_job_spec.rb#L313

My read is that the pattern's purpose is to document which part of the large results context hash these values are coming from. We can change this spec but that will leave it an oddball.